pylint/extensions/_check_docs_utils.py

```diff
@@ -644,7 +644,7 @@ def match_param_docs(self) -> tuple[set[str], set[str]]:
         entries = self._parse_section(self.re_param_section)
         entries.extend(self._parse_section(self.re_keyword_param_section))
         for entry in entries:
-            match = self.re_param_line.match(entry)
+            match = self.re_param_line.match(entry.replace("\\", ""))
```

Comment:
Why don't we change `self._re_param_line`? We can include `\\` there.

In any case, we are also replacing `\\`a bit further down here so that should be fixed as well then if we continue down the current road.

Reply:
I have no idea how to change `_re_param_line`, it's really difficult to read. If you can, plz do paste your regex here!

Reply:
True, I'd rather not touch it as well, but just replacing all `\\` seems error prone as well.
I think it should be as easy as allowing both `\w` and `\\` in `_re_param_line`. You can check this with regex101.com

Reply:
> should be as easy

Oh I tried, I really tried 🤣 , but nothing quite "works". As soon as you try to add some variation of (\\)* or something (Tried lots of ways), it breaks other tests!

Reply:
> In any case, we are also replacing \\a bit further down here so that should be fixed as well then if we continue down the current road.

reasonable and doesn't seem to break tests